lib/rouge/lexers/nial.rb

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+    module Lexers
+        class Nial < RegexLexer
+            title 'Nial'
+            desc 'The Nial programming language (nial-array-language.org)'
+            tag 'nial'
+            filenames '*.ndf', '*.nlg'
+            def self.keywords
```

Comment:
**nitpick:** Could we add an empty line between L10 and L11?

Reply:
added.